docs/specification.md

```diff
@@ -163,9 +164,9 @@ Takes as input new values for all the **configuration options**, these new value
 Called by a `participant` to submit a new proposal.
 Takes as input an IPFS URI and a list of strings for the `n`  available choices.
 
-- fails if the sender does not have a governance token amount of `minimum_balance` or more
 - fails if a proposal for the same IPFS URI was already submitted before
 - fails if the number of choices `n` is `0`
+- calls `[balance_of](https://gitlab.com/tezos/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#balance_of)` on the FA2 governance token with `verify_min_balance` as a callback
```

Comment:
2 tiny suggestions:
1. instead of linking to the `master` branch we should probably point at a specific commit, as tzips can get updated (even tho this one is `Final` and I this link is unlikely to break).
2. You've added another link to this same page below, so I'd make it a reference link.